e2e/client/specs/helpers/authoring.ts

```diff
@@ -450,7 +450,9 @@ class Authoring {
         this.save = function() {
             browser.wait(() => this.save_button.isEnabled(), 2000);
             this.save_button.click();
-            return browser.wait(() => this.save_button.getAttribute('disabled'));
+            return browser.wait(
+                ECE.stalenessOf(this.save_button.element(by.css('[data-test-id="loading-indicator"]'))),
+            );
```

Comment:
it would be good to add explicit timeout, otherwise it uses the default 200s

Reply:
I'd say explicit timeout only makes sense when the test is exceptional and needs more time. Otherwise it'd go with a default timeout - and lower the default if it's too long.

Reply:
problem is it uses the default timeout also for the whole spec, so we can't make it lower

Reply:
By whole `spec` you mean `describe` block? Ok, then we can add an additional limit to browser.wait

Reply:
I think it's for `it` block